packages/relay/tests/lib/sdkClient.spec.ts

```diff
@@ -2045,31 +2067,203 @@ describe('SdkClient', async function () {
       124, 18, 190, 114, 79, 189, 99, 27, 157, 117, 227, 107, 74, 255, 111, 74, 166, 7, 78, 25, 35, 131, 85, 47, 52,
       120, 20,
     ]);
+    const accountId = AccountId.fromString('0.0.1234');
```

Comment:
Should make accountId configurable in case a test runner needs to change the account. The current value is the default

Reply:
Those are only mocked values, we don't do any real requests here